extensions/2.0/Vendor/EXT_mesh_features/README.md

```diff
@@ -616,13 +617,69 @@ Texture filtering must be `9728` (NEAREST), `9729` (LINEAR), or undefined, for a
+| `"UINT8"`                | `5121`                   | `"VEC2"`        | `"VEC2"`               |
+| `"UINT8"`                | `5121`                   | `"VEC3"`        | `"VEC3"`               |
+| `"UINT8"`                | `5121`                   | `"VEC4"`        | `"VEC4"`               |
+| `"UINT8"`                | `5121`                   | `"MAT2"`        | `"MAT2"`               |
+| `"UINT8"`                | `5121`                   | `"MAT3"`        | `"MAT3"`               |
+| `"UINT8"`                | `5121`                   | `"MAT4"`        | `"MAT4"`               |
+| `"UINT16"`               | `5123`                   | `"SINGLE"`      | `"SCALAR"`             |
+| ...                      | ...                      | ...             | ...                    |
+| `"UINT32"`               | `5125` <sup>2</sup>      | `"SINGLE"`      | `"SCALAR"`             |
+| ...                      | ...                      | ...             | ...                    |
+| `"UINT64"`               | `5121` <sup>3</sup>      | `"SINGLE"`      | `"SCALAR"`             |
+| ...                      | ...                      | ...             | ...                    |
+| `"INT8"`                 | `5120` (BYTE)            | `"SINGLE"`      | `"SCALAR"`             |
+| ...                      | ...                      | ...             | ...                    |
+| `"INT16"`                | `5122` (SHORT)           | `"SINGLE"`      | `"SCALAR"`             |
+| ...                      | ...                      | ...             | ...                    |
+| `"INT32"`                | `5124` (INT)<sup>4</sup> | `"SINGLE"`      | `"SCALAR"`             |
+| ...                      | ...                      | ...             | ...                    |
+| `"INT64"`                | `5121` <sup>3</sup>      | `"SINGLE"`      | `"SCALAR"`             |
+| ...                      | ...                      | ...             | ...                    |
+| `"FLOAT32"`              | `5126` (FLOAT)           | `"SINGLE"`      | `"SCALAR"`             |
+| ...                      | ...                      | ...             | ...                    |
+| `"FLOAT64"`              | `5121` <sup>3</sup>      | `"SINGLE"`      | `"SCALAR"`             |
+| ...                      | ...                      | ...             | ...                    |
+| `"BOOLEAN"`              | `5121` <sup>3</sup>      | `"SINGLE"`      | `"SCALAR"`             |
+| `"BOOLEAN"`              | `5121` <sup>3</sup>      | `"ARRAY"`       | `"SCALAR"`<sup>1</sup> |
+| `"STRING"`               | `5121` <sup>3</sup>      | `"SINGLE"`      | `"SCALAR"`             |
+| `"STRING"`               | `5121` <sup>3</sup>      | `"ARRAY"`       | `"SCALAR"`<sup>1</sup> |
+| `"ENUM"`                 | `5121` <sup>3</sup>      | `"SINGLE"`      | `"SCALAR"`             |
+| `"ENUM"`                 | `5121` <sup>3</sup>      | `"ARRAY"`       | `"SCALAR"`<sup>1</sup> |
```

Comment:
For enums `accessor.componentType` would correspond to the `valueType` (INT8, UINT8, INT16, UINT16, INT32, UINT32, INT64, or UINT64)